HISTORY.md

```diff
@@ -0,0 +1,119 @@
+# Changelog
```

Comment:
Since this will be appended to the autogenerated entries, you want to remove these few lines.